Source/ablastr/fields/IntegratedGreenFunctionSolver.cpp

```diff
@@ -38,16 +39,23 @@ void
 computePhiIGF ( amrex::MultiFab const & rho,
                 amrex::MultiFab & phi,
                 std::array<amrex::Real, 3> const & cell_size,
-                amrex::BoxArray const & ba)
+                amrex::BoxArray const & ba,
+                bool const is_2d_slices,
+                bool const is_3d_distributed)
 {
     using namespace amrex::literals;
 
     BL_PROFILE_VAR_NS("ablastr::fields::computePhiIGF: FFTs", timer_ffts);
     BL_PROFILE_VAR_NS("ablastr::fields::computePhiIGF: FFT plans", timer_plans);
     BL_PROFILE_VAR_NS("ablastr::fields::computePhiIGF: parallel copies", timer_pcopies);
+    BL_PROFILE_VAR_NS("ablastr::fields::computePhiIGF: Green functions", timer_Gfunc);
 
     BL_PROFILE("ablastr::fields::computePhiIGF");
```

Comment:
Should we add a check that raises an error if `is_3d_distributed` is selected, but HEFFTE is not available?